sql/src/main/java/io/crate/metadata/Scalar.java

```diff
@@ -79,6 +80,8 @@
     public Symbol normalizeSymbol(Function symbol, TransactionContext transactionContext) {
         try {
             return evaluateIfLiterals(this, symbol);
+        } catch (UnsupportedFeatureException e) {
```

Comment:
There is a note in the commit message. We swallow all exceptions (even Throwables) upon evaluation of literals. This was tricky to debug and I would like to fix that to use checked exceptions instead. For now, I added a special type of exception to indicate that the function actually returned erroneously.